docs/getting-started/connect-widget.mdx

```diff
@@ -13,17 +13,248 @@ Here's a preview:
   width="90%"
   height="700"
   src="https://connect.metriport.com/?token=demo"
-></iframe>
-<br />
+  ></iframe>
+
+
+## How to Implement
+
+You can use the references below for sample implementations on various platforms. Generally, you can simply link to the Connect Widget in web apps, and use WebViews to implement the Widget on mobile platforms.
+
+<Info>
+  As per the [Quickstart guide](/getting-started/connect-quickstart#4-link-to-the-metriport-connect-widget-in-your-app), you'll need to generate a temporary token for each Connect Widget session.
+</Info>
+
+### Web
+
+```typescript
+window.open(`https://connect.metriport.com/?token=${connectToken}`);
+```
+
+<Info>
+  Not all of the data providers support an `iframe` embedding, so we recommend to avoid using an `iframe` for the Connect Widget in your web app.
+</Info>
+
+### iOS (Swift)
+
+```swift
+import SwiftUI
+import Foundation
+import WebKit
+
+@main
+struct Metriport_mobileApp: App {
+    // Return the scene.
+    var body: some Scene {
+        WindowGroup {
+            ContentView()
+        }
+    }
+}
+
+struct ContentView: View {
+    @State private var showWebView = false
+    
+    // var localWebView = MetriportWidget(url: URL(string: "http://localhost:3001/?token={YOUR_TOKEN}")!)
```

Comment:
left-over?

Reply:
This is intentionally left here as an example for local dev